packages/web/src/pages/profile-page/useProfileParams.ts

```diff
@@ -1,21 +1,17 @@
-import { cacheUsersSelectors } from '@audius/common/store'
-import { useSelector } from 'react-redux'
+import { useUserByParams } from '@audius/common/src/api/tan-query/useUserByParams'
```

Comment:
was going on in this page? should get this import fixed in api/index.ts, and then jsut wanted to check that we want to replace the selector with the hook?

Reply:
Yea the import was messed up so I fixed it, but yea from what I was told this type of change is good to do to migrate everything to start using the hooks